docs/index.rst

```diff
@@ -23,4 +23,7 @@ Contents:
    security
    serializer-formatter
    schema-deletion-guidelines
-   maven-plugin
\ No newline at end of file
+   maven-plugin
+
+To see a working example of |sr|, check out our :ref:`Confluent Platform demo<cp-demo>`. The demo shows users how to deploy a Kafka streaming ETL, including Schema Registry, using KSQL for stream processing.
```

Comment:
- revise to remove "our" per style guide.
- revise `The demo shows you how to...`

Reply:
"This demo shows users how to deploy _a_ Kafka streaming ETL" ... the "a" seems out of place. @joel-hamill, is that what you were referring to?